blogs.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Blog Page</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="mystyle.css">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+
+<main role="main">
+    <header class="jumbotron bgColor ">
+      <div class="container bgColor">
+        <img class="logo" src="./images/adoptadog-logo.svg" alt="Logo">
+        <h3 class="slogan">Seattle's premiere dog adoption agency </h3>
+      </div>
+    </header>
+
+    <nav class="marginTop navbar navbar-expand-md navbar-dark bg-dark">
+      <div class="justifyContent container">
+        <a class="navbar-brand" href="index.html">Home   /</a>
+        <a class="navbar-brand" href="./dogs.html">Dogs   /</a>
+        <a class="navbar-brand" href="blogs.html">Blog </a>
+      </div>
+    </nav>
+
+  <main class="main">
+
+    <div class="container">
+      <h2>Adopt A dog Blog</h2>
+      <div class="row marginBottom">
+        <div class="centerImg-md col-lg-4">
+          <img src="./images/blog-1.jpg" alt="Girl With a Dog">
+        </div>
+        <div class="content col-md-12 col-lg-8">
+          <h3>Traveling With your Dog</h3>
+          <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, </p>
+        </div>
+      </div>
+
+      <div class="row marginBottom">
```

Comment:
Each of these div's takes up the whole row, never giving you a 3 wide column. Your strategy for bootstrap isn't quite right.